ivy/src/main/scala/sbt/CustomPomParser.scala

```diff
@@ -95,10 +108,10 @@ object CustomPomParser {
       val mergeDuplicates = IvySbt.hasDuplicateDependencies(md.getDependencies)
 
       val unqualify = toUnqualify(filtered)
-      if (unqualify.isEmpty && extraDepAttributes.isEmpty && !convertArtifacts && !mergeDuplicates)
```

Comment:
Should we kill `mergeDuplicates`, `convertArtifacts` and the `artifactExtIncorrect` method as a part of this PR?